buildkite/resource_cluster_queue.go

```diff
@@ -146,8 +146,15 @@ func (cq *clusterQueueResource) Create(ctx context.Context, req resource.CreateR
 	state.ClusterUuid = types.StringValue(r.ClusterQueueCreate.ClusterQueue.Cluster.Uuid)
 	state.Key = types.StringValue(r.ClusterQueueCreate.ClusterQueue.Key)
 	state.Description = types.StringPointerValue(r.ClusterQueueCreate.ClusterQueue.Description)
-	state.DispatchPaused = plan.DispatchPaused
 
+	// GraphQL API does not allow Cluster Queue to be created with Dispatch Paused
+	if plan.DispatchPaused.ValueBool() {
+		resp.Diagnostics.AddWarning(
+			"Cluster Queue cannot be created with Dispatch Paused",
+			"Setting dispatch_paused in State to 'true' but requires re-run of Apply",
+		)
+	}
+	state.DispatchPaused = plan.DispatchPaused
```

Comment:
To better match the current functionality of the API, I think we should set `state.DispatchPaused = types.BoolValue(true)` here and (modify, but) retain the warning. We should also modify the wording in the Schema  for `dispatch_paused` to state that on resource creation this attribute will be set to `false`.

Alternatively, we can move the logic that calls `pauseDispatchClusterQueue()` and `resumeDispatchClusterQueue()` into its own `func` and call the function from both `Create()` and `Update()`. This would ensure that the "expected" state of a `cluster_queue` resource is correct after each (singular) TF run.

Reply:
Actually, no. Setting `state.DispatchPaused = types.BoolValue(false)` will break Terraform with a `... dispatch_paused: was cty.True, but now cty.False` error if `dispatch_paused = true` on resource creation.